helm-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "safe-transaction-service-helm.fullname" . }}
+  labels:
+    {{- include "safe-transaction-service-helm.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "safe-transaction-service-helm.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "safe-transaction-service-helm.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "safe-transaction-service-helm.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: worker
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["docker/web/celery/worker/run.sh"]
+          env:
+            - name: DATABASE_URL
+              value: postgres://postgres:{{ .Values.global.postgresql.postgresqlPassword}}@{{ .Release.Name }}-postgresql:5432/{{ .Values.global.postgresql.postgresqlDatabase }}
+            - name: ETHEREUM_NODE_URL
+              value: https://forno.celo.org
```

Comment:
This should be a config value as well to make it easy to switch to private forno in the future. Same for all other occurrences of this value.